Tools/nib2cib/NSView.j

```diff
@@ -25,6 +25,7 @@
 
 @import <AppKit/CPView.j>
 
+@class Nib2Cib
```

Comment:
Should not the corresponding line in NSControl.j be removed?

Reply:
You're right. Done in new commit.